Sources/AuthFoundation/Utilities/TimeCoordinator.swift

```diff
@@ -45,24 +45,48 @@ extension Date {
 }
 
 fileprivate var SharedTimeCoordinator: TimeCoordinator = DefaultTimeCoordinator()
-struct DefaultTimeCoordinator: TimeCoordinator {
+class DefaultTimeCoordinator: TimeCoordinator, OAuth2ClientDelegate {
     static func resetToDefault() {
         Date.coordinator = DefaultTimeCoordinator()
     }
     
+    @ThreadSafe
+    private(set) var offset: TimeInterval
+    
+    private var observer: NSObjectProtocol?
+
+    init() {
+        self.offset = 0
+        self.observer = NotificationCenter.default.addObserver(forName: .oauth2ClientCreated,
```

Comment:
Don't see `self.observer.removeObserver` counterpart. You still need to call it

Reply:
D'oh, thanks for catching that. I'm used to using selector-based observers, which automatically remove themselves for you.